### DISCUSSION TITLE
use a List value as a config variable

### DISCUSSION THREAD

**Discussion by dan-sproutward** [postmanlabs/postman-flows#240] [Q&A]
I have a response with an array value that needs to be passed as a config variable to a request. Durable will not take a list. Config variable complains it does not handle lists. 

So I tried to make the entire array value as a string with escaped quotations(\\") and dropping the variable between square brackets to make the array. That has not done the trick but I think I am on the right track. 

I tested this as a string literal plugged directly into the request config and it works. The value looks like this: "account_123", "account_456". 

Now that i know it is possible to create the array, I can't figure out how to create that right format to pass that from the response to the request. Returning this in the response as "\\\"account_123\\\", \\\"account_456\\\"" which stays as a string and when I print to the terminal and looks like this '"googleads_account_123", "googleads_account_456"' but Flow interprets this a a List even though it is not technically a list(no square brackets would be the proper JSON representation).

I have tried variants like escaping the escapes to support double un-escaping which looks like this:  "\\\\\\\"account_123\\\\\\\", \\\\\\\"account_456\\\\\\\"". In the terminal the value looks like this '\"googleads_account_123\", \"googleads_account_456\"' but Flow still interprets this as a List.

BTW, trying to demonstrate escaping is challenging with the preview :).

 Is there some sort of transformation on a JSON array to force the value as a string which will resolve to an array in a request JSON payload when used as a config variable?

**Comment by saswatds** [comment]
Hey @dan-sproutward does the `to_json` operator https://www.postmanlabs.com/postman-flows/docs/operators/ solve your use-case to convert an list to JSON array sting?
